helpers/serverHelpers.js

```diff
@@ -4,24 +4,25 @@ export async function newSession(login, password) {
     const endPoint = SERVER_PATH + '/sessions/login';
     let failedToFetch = false;
 
-    const response = await fetch(
-        endPoint,
-        {
-            method: 'post',
-            body: JSON.stringify({ login, password }),
-            headers: { 'Content-type': 'application/json'}
-        })
-        .catch(() => {
-            failedToFetch = true;
-        });
+    const response = await fetch(endPoint, {
+        method: 'post',
+        body: JSON.stringify({ login, password }),
+        headers: { 'Content-type': 'application/json' },
+    }).catch(() => {
+        failedToFetch = true;
+    });
 
-    if(failedToFetch){
+    if (failedToFetch) {
```

Comment:
Yeees, my eyes are happy now :D

Reply:
I am happy with this too. I thought that it will be harder :)